### DISCUSSION TITLE
Error: 13 when trying to start commissioner

### DISCUSSION THREAD

**Discussion by Glenyz** [openthread/openthread#10693] [Q&A] [answered]
Hi,

I am using the Thread: CLI example provided by Nordic's NCS samples. I have set up the commissioner before and it was working properly, but for some reason now when I try starting it again, I get the following error:

uart:~$ *** Booting nRF Connect SDK 3758bcbfa5cd ***
ot dataset init new
Done
uart:~$ ot dataset commit active
Done
uart:~$ ot ifconfig up
Done
uart:~$ ot thread start
Done
uart:~$ ot commissioner start
Error 13: InvalidState
uart:~$ 

I have not made any changes to the sample or the project configs. Does anyone know why it's giving me this error?

**Comment by jwhui** [comment, marked as answer]
`InvalidState` error indicates that the device is not currently attached to a Thread network:
https://github.com/openthread/openthread/blob/7673194af6c4ede079afc8a9af24f4b0493373a2/src/core/meshcop/commissioner.cpp#L283

You can use the [`state`](https://github.com/openthread/openthread/blob/main/src/cli/README.md#state) CLI command to view the current state.

**Reply by Glenyz** [reply]
I see, I didn't give enough time for the device to attach to a Thread network. Now that I try it again it works fine. Thank you
